django_declarative_apis/machinery/errors.py

```diff
@@ -170,7 +173,7 @@ def __init__(self, additional_info=None):
 class ClientErrorResponseWrapper(ClientError):
     def __init__(self, response):
         error_code = response.status_code
-        error_message = response.content
+        error_message = response.content.decode()
```

Comment:
`error_message` is bytestring unless we do this.  Maybe this wasn't an issue until Python 3?